frontend-project/src/pages/letters/list/index.tsx

```diff
@@ -22,6 +23,11 @@ const TableList: FC<{}> = () => {
       render: (direction: string) =>
         formatMessage({ id: `letters-list.table.direction.${direction.toLowerCase()}` }),
     },
+    {
+      title: formatMessage({ id: 'letters-list.table.columns.audited_institutions.title' }),
```

Comment:
letters-list.table.columns.audited_institution**~s~**.title bo jednak jest to pojedyncza instytucja